src/components/navGlobalStyles.js

```diff
@@ -8,6 +8,9 @@ export const globalStyles = {
     display: "flex",
     flexDirection: "column",
   },
+  a: {
```

Comment:
Generelt skeptisk til å style basert på tag i stedet for klasse, men her vil vi i praksis alltid overstyre selektoren der vi vil det. Klasser er vanskelig å legge til i editoren (for redaktører), så best å slippe å bruke klasse-selektor her.